README.md

```diff
@@ -152,6 +152,11 @@ Headers to load the image with. e.g. `{ Authorization: 'someAuthToken' }`.
 
 ---
 
+### `enableMinificationFilter?: boolean`
```

Comment:
I think it might be better to expose it as an enum, since it seems there are 3 possible values for it, and there could be more added (unlikely but possible)